src/components/Trends/TrendsTable/TrendsTables.js

```diff
@@ -68,17 +97,22 @@ class TrendsTables extends PureComponent {
           isLoggedIn={isLoggedIn || true}
           selectTrend={this.selectTrend}
           selectedTrends={selected}
+          connectedTrends={connectedTrends}
+          trendConnections={trendConnections}
+          connectTrends={this.connectTrends}
+          clearConnectedTrends={this.clearConnectedTrends}
         />
       </div>
     )
   }
 }
 
 const mapStateToProps = ({
-  hypedTrends: { selectedTrends },
+  hypedTrends: { selectedTrends, connectedTrends },
   user: { token }
 }) => ({
   selectedTrends,
+  connectedTrends,
   isLoggedIn: !!token
```

Comment:
@DmitriVanGuard What do you think. Let's use for loggedin one selector function.
Cause if we will change logic of this. There are a lot of changes, we will do.